syntax = "proto2";
package socketprotobuf;

enum mainId
{
	cmd_heart=60000;
	cmd_rcode=65001;
	cmd_security =65002;
}
enum err_code
{
	versionError=1;
	md5Error=2;
	typeError=3;
}

message sc_send_Rcode
{
	required  string rcode=1;
}
message cs_heart_time
{
	required int32  ftime=1;
}

message sc_heart_time
{
	required int32  ftime=1;
}
message cs_md5_security
{
    required int32 id = 1;
    required int32 type = 2;
    required int32 version=3;
	 required string md5=4;
}
message sc_send_sha1
{
	required string sha1=1;
}
